src/com/robibp/trening/EvidencijaDrive.java

```diff
@@ -1,18 +1,23 @@
 package com.robibp.trening;
 
+import java.util.ArrayList;
 import java.util.Scanner;
 
 class EvidencijaDrive {
 
     public static void main(String[] args) {
+        Klinac podatak = new Klinac();
+        UnosKlinca provjeraUnosa = new UnosKlinca();
```

Comment:
Brate nije `provjeraUnosa` nego je `unosKlinca`. 
Onda kasnije imas `provjeraUnosa.unos()` sta ti to govori kad procitas. Provjera unosa pa unos, sta se unosi sta se provjerava ???
`unosKlinca.unos()` e tu onda znas sta se unosi